tests/cli/test_ci.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+
+dnf -y install \
+  anaconda-tui \
+  libgit2-glib \
+  libselinux-python3 \
+  make \
+  pykickstart \
+  python3-coverage \
+  python3-coveralls \
+  python3-flask \
+  python3-gevent \
+  python3-magic \
+  python3-mako \
+  python3-mock \
+  python3-nose \
+  python3-pocketlint \
+  python3-pycdlib \
+  python3-pylint \
+  python3-pyparted \
+  python3-toml \
+  python3-semantic_version \
+  python3-sphinx \
+  python3-sphinx-argparse \
+  python3-sphinx_rtd_theme \
+  python3-rpmfluff \
+  python3-librepo \
+  python3-pyvmomi \
+  beakerlib \
+  sudo \
+  rsync \
+  e2fsprogs \
+  xz-lzma-compat \
+  pbzip2 \
+  squashfs-tools \
+  qemu-img \
+  which 
```

Comment:
@larskarlitski please see my comment above for running `make ci` directly into `test/run`. Possibly lots of the packages above will end up in vm.install which may turn out to be just too much. What do you think ?

Reply:
FTR I'm starting to like the fact that the test script makes sure it has all the packages it needs and this doesn't affect the rest of the test suite (i.e. not in vm.install and affecting all scenarios). I think this is acceptable for now so marking this comment as resolved.